### DISCUSSION TITLE
How to use templates in oh-my-posh

### DISCUSSION THREAD

**Discussion by AviAvinav** [JanDeDobbeleer/oh-my-posh#2968] [Troubleshoot] [answered]
I was trying to create a custom theme, can anyone guide me on what these things are inside templates:

```
"template": " \ue28e {{ .Full }} ",
```

What is `\ue28e` here? Is there a list of these codes somewhere?

**Comment by JanDeDobbeleer** [comment, marked as answer]
@AviAvinav yes, they're [glyphs](https://www.nerdfonts.com/cheat-sheet) inside the nerd font range. Representing an icon.

**Reply by AviAvinav** [reply]
Thanks
